bands/site_sections/band_admin.py

```diff
@@ -51,13 +51,19 @@ def everything(self, out, session):
             'Charity Answer', 'Charity Donation'
         ])
         for band in session.query(Band).all():
+            bio_url = getattr(band.bio, 'pic_url', '')
+            taxes_url = getattr(band.taxes, 'w9_url', '')
+            stageplot_url = getattr(band.stage_plot, 'url', '')
+            absolute_pic_url = build_uber_absolute_url(bio_url) if bio_url else ''
+            absolute_taxes_url = build_uber_absolute_url(taxes_url) if taxes_url else ''
+            absolute_stageplot_url = build_uber_absolute_url(stageplot_url) if stageplot_url else ''
```

Comment:
If we have to repeat this pattern a lot, then ``build_uber_absolute_url`` should probably return an empty string when given a falsey input.